modules/ROOT/pages/deployment/binary/binary-setup.adoc

```diff
@@ -217,33 +217,42 @@ sudo useradd --system --no-create-home --shell=/sbin/nologin ocis
 ----
 
 * We strongly advise *against* using the user `root` for this purpose.
-* Placing the environment file in `{ocis_env}` is only a suggestion, but a good one.
-** The directory `ocis_env` must exist and the user `ocis` must be able to read it.
+* Since `ocis` user doesn't have home directory, these directories `{ocis_env}` and `{ocis_data}` must exist and the user `ocis` must be able to read it.
```

Comment:
```suggestion
* Since the `ocis` user doesn't have a home directory, the directories `{ocis_env}` and `{ocis_data}` must exist and the user `ocis` must be able to read it.
```